tools/src/tester/SchemaValidator.ts

```diff
@@ -32,7 +32,7 @@ export default class SchemaValidator {
   validate (schema: OpenAPIV3.SchemaObject, data: any): Evaluation {
     const validate = this.ajv.compile(schema)
     const valid = validate(data)
-    if (! valid) {
+    if (!valid) {
```

Comment:
Missing lint?

Reply:
Same issue with the previous PR: the rule was moved into a plugin.